### PR TITLE
Update htaccess.txt - Add missing quotation marks

### PR DESCRIPTION
**Pull request by hettiger**

### CODE REVIEW

**Comment by adrianbj** [comment]
I believe this is intentional to make it compatible with Apache 1.3. Are you seeing any issues with it as is?

**Comment by hettiger** [comment]
No it's just that PHPStorm started complaining and it made sense to me. I don't know about that Apache 1.3 compatibility issue.

**Comment by teppokoivula** [comment]
@hettiger, you can see the difference by looking at Apache documentation:

- https://httpd.apache.org/docs/1.3/custom-error.html (1.3, "Prefix the text with a quote")
- https://httpd.apache.org/docs/2.0/custom-error.html (2.0, text instructs to "Prefix the text with a quote" but in examples it is actually *wrapped* with quotes)
- https://httpd.apache.org/docs/2.2/custom-error.html (2.2, text has been corrected to "Wrap the text with quotes")

In other words: this is a feature and PHPStorm is at fault here.. or, rather, it probably doesn't know anything about how Apache configuration works :)

**Comment by hettiger** [comment]
@teppokoivula Thank you for clarification. A good reason to make pull requests is that you can  learn from it i guess :-)
